package.json

```diff
@@ -7,13 +7,15 @@
     "@testing-library/jest-dom": "^5.17.0",
```

Comment:
- [ ] It would be better if we create the MIT license file in our project and customize it accordingly. I leave the comment here because in the file changes i did not find the README file in here

Reply:
Done @addod19.